test/index.html

```diff
@@ -3,18 +3,19 @@
 
 <head>
   <title>Backbone.Stickit Test Suite</title>
+  <meta charset="UTF-8">
 
-	<link rel="stylesheet" href="vendor/qunit.css" type="text/css"/>
+	<link rel="stylesheet" href="vendor/qunit-2.0.1.css" type="text/css"/>
 
 	<!--script type="text/javascript" src="http://cdnjs.cloudflare.com/ajax/libs/zepto/1.1.1/zepto.min.js"></script>
 	<script type="text/javascript" src="vendor/zepto-data.js"></script-->
-	<script type="text/javascript" src="vendor/jquery.js"></script>
-	<script type="text/javascript" src="vendor/qunit.js"></script>
+	<script type="text/javascript" src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
+	<script type="text/javascript" src="vendor/qunit-2.0.1.js"></script>
```

Comment:
Please split this change into another PR. And replace the file in place instead of creating a new one?